examples/prism-promql.html

```diff
@@ -0,0 +1,18 @@
+<h2>Examples</h2>
+<pre><code>
+# These examples are taken from: https://prometheus.io/docs/prometheus/latest/querying/examples/
```

Comment:
Otherwise, the code block will start with an empty line.

```suggestion
<pre><code># These examples are taken from: https://prometheus.io/docs/prometheus/latest/querying/examples/
```